xbmc/platform/darwin/tvos/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(SOURCES 
+			MainApplication.mm
```

Comment:
Remove this line. It ends in duplicate symbol errors as its built/linked elsewhere already.

Reply:
Should I remove MainApplication.h too?

Reply:
No, just the mm